src/styles/blocks/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background: $main-background-color;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+
+    display: block;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    margin-bottom: 4px;
+
+    color: $color-main-accent;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    margin-bottom: 16px;
+
+    color: $color-secondary;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-row {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-paragraph {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $color-secondary;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+
+    color: $color-main-accent;
+  }
+
+  &__buy {
+    height: 40px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    background: $color-accent-blue;
+    border-radius: 5px;
+    border: 1px solid $color-accent-blue;
+
+    color: #fff;
+    transition: all 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```